pkgs/development/ocaml-modules/sedlex/default.nix

```diff
@@ -1,33 +1,32 @@
-{ stdenv, fetchzip, ocaml, findlib, gen, ppx_tools_versioned, ocaml-migrate-parsetree }:
+{ lib
+, fetchzip
+, buildDunePackage
+, gen
+, ocaml-migrate-parsetree
+, ppx_tools_versioned
+}:
 
-if !stdenv.lib.versionAtLeast ocaml.version "4.02"
-then throw "sedlex is not available for OCaml ${ocaml.version}"
-else
+buildDunePackage rec {
+  pname = "sedlex";
+  version = "2.1";
 
-stdenv.mkDerivation rec {
-  name = "ocaml${ocaml.version}-sedlex-${version}";
-  version = "1.99.5";
+  minimumOCamlVersion = "4.02.3";
 
   src = fetchzip {
-    url = "https://github.com/ocaml-community/sedlex/archive/fb84e1766fc4b29e79ec40029ffee5cdb37b392f.tar.gz";
-    sha256 = "0phnqyn6mpv5byr1kkphl24y9q9fb2k3xg9yb457h5816q6ya72n";
+    url = "https://github.com/ocaml-community/sedlex/archive/v${version}.tar.gz";
+    sha256 = "05f6qa8x3vhpdz1fcnpqk37fpnyyq13icqsk2gww5idjnh6kng26";
   };
 
-  buildInputs = [ ocaml findlib ];
+  propagatedBuildInputs = [
+    gen ocaml-migrate-parsetree ppx_tools_versioned
+  ];
 
-  propagatedBuildInputs = [ gen ocaml-migrate-parsetree ppx_tools_versioned ];
-
-  buildFlags = [ "all" "opt" ];
-
-  createFindlibDestdir = true;
-
-  dontStrip = true;
+  doCheck = true;
 
   meta = {
     homepage = https://github.com/ocaml-community/sedlex;
```

Comment:
You need to add `"` to the homepage. Please :)

Reply:
Done.